swatch-producer-aws/src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 SERVER_PORT=${clowder.endpoints.swatch-producer-aws.port:8000}
 LOGGING_LEVEL_COM_REDHAT_SWATCH=INFO
 LOGGING_LEVEL_ROOT=INFO
-SWATCH_INTERNAL_SUBSCRIPTION_ENDPOINT=http://${clowder.endpoints.capacity-ingress.hostname}:${clowder.endpoints.capacity-ingress.port}
+SWATCH_INTERNAL_SUBSCRIPTION_ENDPOINT=${clowder.endpoints.swatch-subscription-sync-service.url}
```

Comment:
The capacity-ingress didn't exist. The SWATCH_INTERNAL_SUBSCRIPTION_ENDPOINT value was being set in the clowderapp.yaml.
After these changes, all the Quarkus-based services will populate the configuration using the clowder config source.